docs/components/sidecar.md

```diff
@@ -29,8 +29,9 @@ Prometheus servers connected to the Thanos cluster via the sidecar are subject t
 * The `--web.enable-admin-api` flag is enabled to support sidecar to get metadata from Prometheus like external labels.
 * The `--web.enable-lifecycle` flag is enabled if you want to use sidecar reloading features (`--reload.*` flags).
 
-If you choose to use the sidecar to also upload to object storage:
+If you choose to use the sidecar to also upload uncompacted prometheus blocks(for compacted blocks, see [Upload compacted blocks](./sidecar.md/#upload-compacted-blocks-experimental)) to object storage:
```

Comment:
I think this is too much information for one sentence; it is a little confusing to read IMO even for people who know how it works :)
Try splitting it up?